PGIF.py

```diff
@@ -56,8 +56,8 @@ def setup(self):
             except:
                 print('ERROR: System could not read GIF Image(s)')
             else:
-                image = pygame.transform.scale(image, self.scale_to)
+                image = pygame.transform.smoothscale(image, self.scale_to)

```

Comment:
I was not aware of smooth-scale function. It is a good idea to smooth-scale in the future use. However, it was not necessary to use in my case.